scripts/exglobal_prep_land_obs.py

```diff
@@ -17,7 +17,10 @@
 
     # Take configuration from environment and cast it as python dictionary
     config = cast_strdict_as_dtypedict(os.environ)
+    HH = os.environ.get("CDATE")[8:]
```

Comment:
```suggestion
```
There is no need to go to `os.environ` after creating `config` from the environment.

Reply:
Done as suggested.